### PR TITLE
[OSDOCS-5514]: Add hosted control planes book

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.13

Issue: https://issues.redhat.com/browse/OSDOCS-5514

Link to docs preview: https://57076--docspreview.netlify.app/openshift-enterprise/latest/hosted_control_planes/index.html

QE review:
- [x] QE has approved this change.

Additional information: This PR adds a new book to the TOC that will contain content about hosted control planes. Additional assemblies and modules will be added to the book in separate PRs.

### CODE REVIEW

**Comment by jiezhao16** [comment]
lgtm

File: hosted_control_planes/index.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="hcp-overview"]
+= Hosted control planes overview
+include::_attributes/common-attributes.adoc[]
+:context: hcp-overview
+
+You can deploy {product-title} clusters by using two different control plane configurations: standalone or hosted control planes. The standalone configuration uses dedicated virtual machines or physical machines to host the control plane. With hosted control planes for {product-title}, you create control planes as pods on a hosting cluster without the need for dedicated physical machines for each control plane.
```
Comment (jeana-redhat) [inline hosted_control_planes/index.adoc:7]: In this line, do dedicated _virtual_ machines need a mention too? I could easily just be misunderstanding though :)
```suggestion
You can deploy {product-title} clusters by using two different control plane configurations: standalone or hosted control planes. The standalone configuration uses dedicated virtual machines or physical machines to host the control plane. With hosted control planes for {product-title}, you create control planes as pods on a hosting cluster without the need for dedicated virtual or physical machines for each control plane.
```

File: hosted_control_planes/index.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/clusters/cluster_mce_overview#hypershift-addon-intro[HyperShift add-on (Technology Preview)]
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/clusters/cluster_mce_overview#hosted-control-planes-intro[Hosted control planes for Red Hat OpenShift Container Platform (Technology Preview)]
+
+
+include::modules/hosted-control-planes-version-support.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* For more information about the `HostedCluster` and `NodePool` resources, see the link:https://hypershift-docs.netlify.app/reference/api/[HyperShift API Reference].
```
Comment (jeana-redhat) [inline hosted_control_planes/index.adoc:24]: Additional resources are bare links, no intro text. Also please verify this is the link you want to merge.
```suggestion
* link:https://hypershift-docs.netlify.app/reference/api/[HyperShift API Reference].
```

**Review by jeana-redhat — COMMENTED** [review]
Great start, and really appreciate that you are adding this part first as a framework. Just had a couple questions and one little formatting thing.

File: hosted_control_planes/index.adoc (hunk 1 above)
Comment (lahinson) [reply hosted_control_planes/index.adoc]: Good catch! Yes, it should say "virtual or physical". Fixed ✅

File: hosted_control_planes/index.adoc [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/clusters/cluster_mce_overview#hosted-control-planes-intro[Hosted control planes for Red Hat OpenShift Container Platform (Technology Preview)]
+
+
+include::modules/hosted-control-planes-version-support.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* link:https://hypershift-docs.netlify.app/reference/api/[HyperShift API Reference].
```
Comment (skrthomas) [inline hosted_control_planes/index.adoc:25]: I think you can remove this period

**Review by skrthomas — COMMENTED** [review]
A teeny nit but LGTM otherwise.

File: hosted_control_planes/index.adoc (hunk 3 above)
Comment (lahinson) [reply hosted_control_planes/index.adoc:25]: Good catch!

**Pull request merged**